-code-checks

ERROR: Build errors found during clang-tidy run.
```
JetMETCorrections/Type1MET/src/METCorrectionAlgorithm.cc:71:37: error: out-of-line definition of 'compMETCorrection' does not match any declaration in 'METCorrectionAlgorithm' [clang-diagnostic-error]
CorrMETData METCorrectionAlgorithm::compMETCorrection(edm::Event& evt) {
                                    ^
Suppressed 774 warnings (773 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/JetMETCorrections/Type1MET/interface/METCorrectionAlgorithm.h
+++ b/JetMETCorrections/Type1MET/interface/METCorrectionAlgorithm.h
@@ -34,7 +34,7 @@ public:
   explicit METCorrectionAlgorithm(const edm::ParameterSet&, edm::ConsumesCollector&& iConsumesCollector);
   ~METCorrectionAlgorithm();
 
-  CorrMETData compMETCorrection(edm::Event&, const edm::EventSetup&);
+  CorrMETData compMETCorrection(edm::Event&);
 
 private:
   typedef std::vector<edm::InputTag> vInputTag;

